releases/release-1.22/README.md

```diff
@@ -0,0 +1,110 @@
+* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
+* [Internal Contact Info][Internal Contact Info] (accessible only to members of [kubernetes-release-team@])
+
+#### Tracking docs
+
+* Enhancements Tracking Sheet: TODO
+* Bug Triage Tracking Sheet: TODO
+* CI Signal Report: TODO
+* [Retrospective Document][Retrospective Document]
+* [kubernetes/sig-release v1.22 milestone](https://github.com/kubernetes/kubernetes/milestone/50)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## TL;DR
+
+The 1.22 release cycle is proposed as follows:
+
+- **Monday, April 26th**: Week 1 - Release cycle begins
+- **Tuesday, May 25th**: Week 5 - [Enhancements Freeze](../release_phases.md#enhancements-freeze)
+- **Tuesday, July 6th**: Week 11 - [Code Freeze](../release_phases.md#code-freeze)
+- **Tuesday, July 13th**: Week 12 - [Test Freeze](../release_phases.md#test-freeze)
+- **Wednesday, July 27th**: Week 14 - Docs must be completed and reviewed
+- **Tuesday, August 3rd**: Week 15 - Kubernetes v1.22.0 released
+- **TBD**: Week 16 - Release Retrospective
+
+## Timeline
+
+| **What** | **Who** | **When** | **WEEK** | **CI SIGNAL** |
+|---|---|-------|---|---|
+| Start of Release Cycle | Lead | Mon April 26 | week 1 | [master-blocking] |
+| Start Enhancements Tracking | Enhancements Lead | Tue April 27 | week 1 | |
+| 1.22.0-alpha.1 released | Branch Manager (TBD) | TBD  | TBD | |
+| Schedule finalized | Lead | Thurs April 29 | week 1 | |
+| Team finalized | Lead | Friday April 30 | week 1 | |
+| 1.22.0-alpha.2 released | Branch Manager (TBD) | TBD | TBD | |
+| KubeCon EU + Co-located events | | May 3-7 | | |
+| **Begin [Enhancements Freeze]** (Time TBD) | Enhancements Lead | Tue May 25 | week 5 | [master-blocking], [master-informing] |
+| 1.22.0-alpha.3 released | Branch Manager (TBD) | TBD | TBD | |
+| 1.22.0-beta.0 released | Branch Manager (TBD) | TBD | TBD | |
+| **Begin [Burndown]** (MWF meetings) | Lead | Mon June 28 | week 10 | [1.22-blocking], [master-blocking], [master-informing] |
+| **Call for [Exceptions][Exception]** | Lead | Mon June 28 | week 10 | |
+| Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | TBD | TBD | |
+| **Begin Feature blog freeze** | Comms Lead | TBD | TBD | |
+| 1.22.0-beta.1 released | Branch Manager (TBD) | TBD | TBD | |
+| **Begin [Code Freeze]** (Time TBD) | Branch Manager | Tue July 6 | week 11 | |
+| Burndown Meetings daily| Lead | Tue July 6 | week 11 | |
+| Docs deadline - Open placeholder PRs | Docs Lead | Fri July 9 | week 11 | |
+| **[Test Freeze]** (Time TBD) | Branch Manager | Tue July 13 | week 12 | |
+| Docs deadline - PRs ready for review | Docs Lead | Tue July 20 | week 13 | |
+| 1.22.0-rc.0 released | Branch Manager (TBD) | TBD | TBD | |
```

Comment:
This was raised before and I'll raise that again. It would be good to have beta.2 release a couple of days after code freeze, when all PRs merge. This approach allows downstream consumers to use the release payload sooner and provide feedback about the overall state of the release. I'd really like to see that being called out explicitly in the timeline.

Reply:
Let's do it. Just one detail:

The schedule we put together for 1.21 was tight timewise and would have been hard to squeeze another beta in. Some of the dates for 1.22 are still TBD but if we end up in the same situation where the weeks are too narrow we could perhaps decouple code freeze from beta.1. We freeze the branch, wait about a week, _and then cut beta.1_.  I don't know if there is a problem with that strategy I'm not seeing.

OTOH If the weeks around that time allow for it, we can cut a `beta.2`, no problem.

Reply:
@soltysh: added a TBD deadline for the beta.2 release cut

Reply:
@soltysh -- Thank you for bringing this up here as well. We did discuss it during the 1.21 retrospective and found this to be adding a lot of value. 👍🏽 

We had decided to put in a release cut (can be called beta.2 or rc.0, whatever is suitable) a few days post-Code Freeze.

@puerco -- We can ponder on the idea of decoupling Code Freeze from beta.1. Perhaps, maybe cut beta.1 the day after Code Freeze is put in?